src/ExpandObjects/epfilter.f90

```diff
@@ -14391,7 +14393,7 @@ SUBROUTINE CreateNewUnitary
   CALL CreateNewObj('SetpointManager:SingleZone:Cooling')
   CALL AddToObjFld('Name', base + usAirHandlerNameOff,' Economizer Supply Air Temp Manager')
   CALL AddToObjStr('Control Variable','Temperature')
-  CALL AddToObjStr('minimum supply air temperature {C}','13')
+  CALL AddToObjFld('minimum supply air temperature {C}',base + usCoolDesignSupplyTempOff,'')
   CALL AddToObjStr('maximum supply air temperature {C}','45')
```

Comment:
This is the cooling setpoint manager, so keep the fixed upper limit. Basically, this allows the cooling setpoint to go high enough to turn off the cooling coil if there's not cooling load.